Jedi analysis

stephen-asare opened the pull request:
<!--
Project Pythia has automated review requests. If you are not ready for reviews on this contribution, please
open this Pull Request as a "Draft" by clicking the dropdown on the green "Create Pull Request" button in the
lower right corner here.
-->

guoqing-noaa commented:
@stephen-asare Thanks for the PR. Could you explain more on what changes you have made in this PR?
It looks like many changes in this PR have already been merged into the `main` branch.

Could you do as follows to make sure you add new changes to the top of the `main` branch?
```
1. sync the main branch of your fork to the latest commit of the authoritative ProjectPythia repo.
2. under you Jupyter Lab:
git stash
git checkout main
git pull
git checkout -b newPR
```
Run the `jedi-mpas.ipynb` first, and then check what changes you would like to make. Thanks!

stephen-asare commented:
> @stephen-asare Thanks for the PR. Could you explain more on what changes you have made in this PR? It looks like many changes in this PR have already been merged into the `main` branch.
> 
> Could you do as follows to make sure you add new changes to the top of the `main` branch?
> 
> ```
> 1. sync the main branch of your fork to the latest commit of the authoritative ProjectPythia repo.
> 2. under you Jupyter Lab:
> git stash
> git checkout main
> git pull
> git checkout -b newPR
> ```
> 
> Run the `jedi-mpas.ipynb` first, and then check what changes you would like to make. Thanks!



> @guoqing-noaa Okay. I will do that as soon as possible.

In this PR I updated my visualizations on the increments and removed duplicated codes to make it easier to read. Additionally, I made the contours simplified the function for the color bar and added more options. I also just realized my PR #40 is resolved but the code in the main Pythia repo was behind mine so I wanted to update it.
